jwst/photom/photom.py

```diff
@@ -239,9 +239,8 @@ def calc_nirspec(self, ftab, area_fname):
                         self.input.err /= sens2d
                         self.input.var_poisson /= sens2d**2
                         self.input.var_rnoise /= sens2d**2
-                        if (self.input.var_flat is not None and
-                            np.size(self.input.var_flat) > 0):
-                                self.input.var_flat /= sens2d**2
+                        if (self.input.var_flat is not None and np.size(self.input.var_flat) > 0):
```

Comment:
The outer parentheses are no longer needed.  I only included them so that the line could be broken without using a backslash.